client/views/signup.jsx

```diff
@@ -0,0 +1,75 @@
+/** @jsx React.DOM */
+var Login = React.createClass({
+  
+  mixins: [Backbone.Events],
+
+  getInitialState: function() {
+    return new UserModel(); 
+  },
+
+  //Set listener on state (which is a backbone model)
+  componentDidMount: function(){
+    this.listenTo(this.state, 'change', function(){
+      console.log('heard a change event!');
+    }, this);
+  },
+
+  //Every time a user interacts with the form, we need to update the state of the view to reflect that change.
+  handleChange: function(event) {
```

Comment:
Do we need to change the model on every change event? Could we just update the model when the user submits?